workshops/models.py

```diff
@@ -227,9 +228,14 @@ def ongoing_events(self):
         return queryset
 
     def unpublished_events(self):
-        '''Return a queryset for events that are not yet published.'''
+        '''Return events without URLs that are upcoming or have unknown starts.
```

Comment:
It might make more sense to change the name from "unpublished" to 
"incomplete" to better reflect the actual semantics. If so, should I put 
that into this PR or do it separately?